src/mailroom.py

```diff
@@ -0,0 +1,99 @@
+"""Code to create email and report from a list of donors."""
+donors = []
+
+
+class Donor:
+    """Stores donor info."""
+
+    first_name = ""
+    last_name = ""
+    donations = 0
+    total_amount = 0
+
+    @classmethod
```

Comment:
Interesting, but the typical way of doing this is just using __int__(self, first_name, etc....):